app/assets/javascripts/main.js

```diff
@@ -1,27 +1,56 @@
 $(document).ready(function() {
-  $('textarea').elastic();
   $('form.new_job_post_request, form.new_job, form.edit_job, #sign_in form').superLabels({
```

Comment:
Other than the jQuery version of Rails UJS, `superlables` is the only thing left from what I can tell that relies on jQuery.